docs/index.html

```diff
@@ -307,6 +307,35 @@ <h2>Terminology</h2>
 
     <section id="conformance"></section>
 
+    <section class="informative">
+      <h2>Interoperability Testing</h2>
+      <p>
+        Participating platforms are proving actual interoperability by enrolling in continuous integration tests. These are based on Postman 
+        collections which orchestrate the interaction of participating parties; for example, `Credential Issuance`, `Credential Verification`, 
+        `Presentation Exchange`, etc. 
+        These Postman collections are described with tutorials here: https://github.com/w3c-ccg/traceability-interop/tree/main/docs/tutorials
+      </p>
+      <p>
+        An interoperability report is continuously created from the results of executing these collections, targeting the participating 
+        platforms. In cases of multi-party interactions (such as party A making a verifiable presentation to party B), all combinations of 
+        platform interchange are executed. 
+        The execution of the continuous interoperability report is done with GitHub Actions on the Traceability Interoperability GitHub 
+        repository: https://github.com/w3c-ccg/traceability-interop/actions/workflows/interoperability-report.yml
+      </p>
+      <p>
+        In each test, the GitHub Action runner carries out a number of invocations, acting on behalf of varying parties taking different roles 
+        (`issuer`, `holder` and `verifier`), in turn invoking the participating platform. 
+        Below is an example of calls made by the runner as it is executing a DIDAuth Presentation Exchange flow:
+        <ul>
+          <li>Runner acts as party A in the role of `issuer`, calling `/credentials/issue` on platform X
```

Comment:
```suggestion
          <li>Runner calls `/credentials/issue` on Party A's platform, allowing Party A to act as the issuer.
```

Reply:
I think similar changes across the board going down the list

Reply:
I like this, @mkhraisha. "Party A's platform" reads much clearer than "Platform X".